code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -97,13 +97,13 @@
 		to_chat(src, SPAN_WARNING("You don't possess an innate healing ability."))
 		return
 	if(!aura.can_toggle())
-		to_chat(src, SPAN_WARNING("You can't toggle the healing at this time!"))
+		to_chat(src, SPAN_WARNING("Your body can't shift its processes at this time!"))
 		return
 	aura.toggle()
 	if (aura.innate_heal)
-		to_chat(src, "<span class='alium'>You are now using nutrients to regenerate.</span>")
+		to_chat(src, "<span class='alium'>You shift your body's processes to use nutrients to regenerate.</span>")
 	else
-		to_chat(src, "<span class='alium'>You are no longer using nutrients to regenerate.</span>")
+		to_chat(src, "<span class='alium'>You shift your body's processes to no longer use nutrients to regenerate.</span>")
```

Comment:
Use macros :trollface: 
I'd also like the message to clarify that it is used for healing, "shift its processes" on line 100 is rather vague. Other lines are okay.

Reply:
There's no macro for alium 
![Screenshot_2021-09-29_222746](https://user-images.githubusercontent.com/25853190/148673057-68ea4902-8d48-4f94-a0e4-82e23369bef7.png)

Will do on the message change.

Reply:
> No macro
Then add one :thinking:

Reply:
done